mlflow/keras/autologging.py

```diff
@@ -209,7 +209,7 @@ def _patched_inference(original, inst, *args, **kwargs):
         log_fn_args_as_params(original, [], kwargs, unlogged_params)
         model_id = None
         if log_models:
-            model_id = mlflow.initialize_logged_model("model").model_id
+            model_id = mlflow.initialize_logged_model("model", flavor="keras").model_id
```

Comment:
nit: shall we define FLAVOR_NAME for consistency?

Reply:
Then, shall we move the const in this file or another file and import `FLAVOR_NAME` in `mlflow/keras/init.py`?  We're doing a similar measure in other flavors such as dspy or langchain.